pkg/flink/internal/store/store_utils_test.go

```diff
@@ -131,12 +131,20 @@ func TestProcessResetStatement(t *testing.T) {
 	}
 	s := NewStore(client, nil, &appOptions, tokenRefreshFunc).(*Store)
 	defaultSetOutput := createStatementResults([]string{"Key", "Value"}, [][]string{
-		{config.ConfigKeyCatalog, fmt.Sprintf("%s (default)", appOptions.EnvironmentName)},
```

Comment:
where do appOptions come from? In theory it is fine if we set this here, but they must be correct. and this is what caused issues.

Reply:
appOptions are defined according to what the user passes when starting the `flink shell`. 

Environment is a required parameter, so we'll always intialize the catalog for the user when starting the shell. But now he's able to reset it.

Database is not required, but I'll be initialized if the user passes the "--database" parameter to `flink shell`, or else it will be nil. It can also be reset now.

Reply:
appOptions is what you pass via command line flags. So if you want this to reset to null, we need to use them only once

Reply:
thanks for the explanation, let's give it a try :)